src/taipy/gui/gui.py

```diff
@@ -1493,6 +1493,9 @@ def run(
         # Load application config from multiple sources (env files, kwargs, command line)
         self._config._build_config(run_root_dir, self.__env_filename, kwargs)
 
+        if app_config["debug"]:
+            async_mode = "threading"
```

Comment:
Add a warning if async_mode is not threading?

Reply:
You mean if it is forced to something else?
That would make sense...